src/EventStore.Core/Authorization/LegacyStreamPermissionAssertion.cs

```diff
@@ -25,6 +30,26 @@ public class
 
 		public ValueTask<bool> Evaluate(ClaimsPrincipal cp, Operation operation, PolicyInformation policy,
 			EvaluationContext context) {
+			var startedAt = _sw.Elapsed;
+			var authorize = false;
+			if (!_allowAnonymousStreamAccess) {
+				var claims = cp.Claims;
+				
+				foreach (var claim in claims) {
+					if (claim.Type != ClaimTypes.Name) continue;
+					authorize = true;
+					break;
+				}
+
+				if (!authorize) {
+					Log.Warning(
```

Comment:
im not sure we need the warning and stopwatch in here, the ~same message will be logged in the PolicyAuthorizationProvider i think?

Reply:
actually scratch that, i dont think we should be changing this class at all, i'll push what i mean